expyfun/tests/test_experiment_controller.py

```diff
@@ -550,10 +553,12 @@ def test_mouse_clicks(hide_window):
 
 
 @requires_opengl21
+@pytest.mark.timeout(30)
 def test_background_color(hide_window):
     """Test setting background color"""
     with ExperimentController(*std_args, participant='foo', session='01',
                               output_dir=None, version='dev') as ec:
+        print((ec.window.width, ec.window.height))
```

Comment:
is this cruft or meant to be part of the test?

Reply:
Mostly cruft, doesn't hurt since pytest captures output, but I'll get rid of it since we don't really need it